newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#define N 8
+
+isNumberInOrder(int[], int, int); /*Declaration of a function*/
+
+int main1()
+{
+	int arr[N], number, i, flag = 1, counter = 0;
+
+	while (flag)
+	{
+		printf("Enter %d integer numbers, please: ", N);
```

Comment:
and what I'd the user enters the number 300? Mention that the input should be digits from 0 to 9